<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DispContDlg</class>
 <widget class="QWidget" name="DispContDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>208</width>
    <height>542</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>208</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KawaiiGL Control</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>400</height>
      </size>
     </property>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>85</width>
        <height>3</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="SD_points">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>137</y>
        <width>71</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Points</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="coordFontSize">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>249</y>
        <width>44</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>6</number>
      </property>
      <property name="maximum">
       <number>72</number>
      </property>
      <property name="value">
       <number>8</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>46</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Round</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="SD_litback">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>65</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Lit backface</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="SD_polygons">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>233</y>
        <width>71</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Polygons</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>26</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <widget class="QCheckBox" name="SD_pass1">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>4</y>
         <width>66</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Pass 1</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="SD_p1r">
       <property name="geometry">
        <rect>
         <x>61</x>
         <y>4</y>
         <width>16</width>
         <height>18</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QCheckBox" name="SD_light">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>5</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Light</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>119</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QCheckBox" name="SD_pass5">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>4</y>
         <width>66</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Pass 5</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="SD_p5r">
       <property name="geometry">
        <rect>
         <x>61</x>
         <y>5</y>
         <width>21</width>
         <height>18</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>102</x>
        <y>130</y>
        <width>85</width>
        <height>3</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="SD_cull">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>84</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Cull faces</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="SD_lines">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>195</y>
        <width>71</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Lines</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="SD_persp">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>45</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Perspective</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>96</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QCheckBox" name="SD_pass4">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>4</y>
         <width>66</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Pass 4</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="SD_p4r">
       <property name="geometry">
        <rect>
         <x>61</x>
         <y>5</y>
         <width>21</width>
         <height>18</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QCheckBox" name="coordNum">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>252</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Coordinates</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="SD_lightMove">
      <property name="geometry">
       <rect>
        <x>119</x>
        <y>25</y>
        <width>71</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Move</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="SD_unused">
      <property name="geometry">
       <rect>
        <x>115</x>
        <y>157</y>
        <width>71</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Unused</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>73</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QCheckBox" name="SD_pass3">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>4</y>
         <width>66</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Pass 3</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="SD_p3r">
       <property name="geometry">
        <rect>
         <x>61</x>
         <y>4</y>
         <width>21</width>
         <height>18</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QCheckBox" name="SD_linesAll">
      <property name="geometry">
       <rect>
        <x>115</x>
        <y>214</y>
        <width>61</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Subdiv</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resetLight">
      <property name="geometry">
       <rect>
        <x>166</x>
        <y>24</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="coordName">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>253</y>
        <width>16</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>49</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QCheckBox" name="SD_pass2">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>4</y>
         <width>66</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Pass 2</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="SD_p2r">
       <property name="geometry">
        <rect>
         <x>61</x>
         <y>5</y>
         <width>21</width>
         <height>18</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QCheckBox" name="SD_vtxNormals">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>154</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Smooth</string>
      </property>
     </widget>
     <widget class="QPushButton" name="selColBot">
      <property name="geometry">
       <rect>
        <x>62</x>
        <y>321</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Selected</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backColBot">
      <property name="geometry">
       <rect>
        <x>124</x>
        <y>297</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Background</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mattColBot">
      <property name="geometry">
       <rect>
        <x>62</x>
        <y>297</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Default</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="shineEdit">
      <property name="geometry">
       <rect>
        <x>79</x>
        <y>348</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Shininess</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ambBot">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>297</y>
        <width>51</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Ambient</string>
      </property>
     </widget>
     <widget class="QPushButton" name="diffBot">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>321</y>
        <width>51</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Diffuse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="specBot">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>345</y>
        <width>51</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Specular</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>280</y>
        <width>46</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Light 0:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>67</x>
        <y>281</y>
        <width>46</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Material:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveBot">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>195</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Save the subdivided object to a file</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="lineColBot">
      <property name="geometry">
       <rect>
        <x>144</x>
        <y>321</y>
        <width>51</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Lines</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="SD_pntLight">
      <property name="geometry">
       <rect>
        <x>152</x>
        <y>5</y>
        <width>72</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Pnt</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="SD_triang">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>175</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Triangulate</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="blendBox">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>104</y>
        <width>72</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Blend</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="SD_allpoly">
      <property name="geometry">
       <rect>
        <x>115</x>
        <y>176</y>
        <width>72</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>All Poly</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="pointSizeSpin">
      <property name="geometry">
       <rect>
        <x>155</x>
        <y>135</y>
        <width>42</width>
        <height>21</height>
       </rect>
      </property>
      <property name="accelerated">
       <bool>true</bool>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="value">
       <number>14</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="zbufBox">
      <property name="geometry">
       <rect>
        <x>149</x>
        <y>104</y>
        <width>72</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Z-buf</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resetBot">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>221</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset All</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resetView">
      <property name="geometry">
       <rect>
        <x>72</x>
        <y>221</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>23</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Reset View</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QSlider" name="clipSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>376</y>
        <width>181</width>
        <height>19</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="lineWidthSpin">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>195</y>
        <width>42</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="linePoly">
      <property name="geometry">
       <rect>
        <x>163</x>
        <y>234</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>line</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="texSel"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="tex0cont" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>0.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tex1NameLabel"/>
      </item>
      <item>
       <widget class="QPushButton" name="tex1FileBot">
        <property name="maximumSize">
         <size>
          <width>21</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>label_3</zorder>
     <zorder>tex1FileBot</zorder>
     <zorder>tex1NameLabel</zorder>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="tex1cont" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>1.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tex2NameLabel"/>
      </item>
      <item>
       <widget class="QPushButton" name="tex2FileBot">
        <property name="maximumSize">
         <size>
          <width>21</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>2.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tex3NameLabel"/>
     </item>
     <item>
      <widget class="QPushButton" name="tex3FileBot">
       <property name="maximumSize">
        <size>
         <width>21</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>3.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tex4NameLabel"/>
     </item>
     <item>
      <widget class="QPushButton" name="tex4FileBot">
       <property name="maximumSize">
        <size>
         <width>21</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>SD_light</sender>
   <signal>toggled(bool)</signal>
   <receiver>SD_lightMove</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>5</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SD_points</sender>
   <signal>toggled(bool)</signal>
   <receiver>SD_unused</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
